scripts/generate-env-example-networks.mjs

```diff
@@ -2,13 +2,14 @@ import { networkName, getRpcUrlEnvVar } from '@blocksense/base-utils/evm';
 import { kebabToScreamingSnakeCase } from '@blocksense/base-utils/string';
 
 const res = networkName.literals
-  .map(
-    n =>
-      `# ${n}` +
-      '\n' +
-      `${getRpcUrlEnvVar(n)}="<RPC_URL>"` +
-      '\n' +
-      `EXTRA_SIGNERS_${kebabToScreamingSnakeCase(n)}="<EXTRA_SIGNERS>"`,
+  .map(n =>
+    [
+      `# ${n}`,
+      `${getRpcUrlEnvVar(n)}="<RPC_URL>"`,
+      `REPORTER_ADDRESSES_${kebabToScreamingSnakeCase(n)}="<REPORTERS>"`,
+      `ADMIN_EXTRA_SIGNERS_${kebabToScreamingSnakeCase(n)}="<EXTRA_SIGNERS>"`,
+      `${kebabToScreamingSnakeCase(n)}_API_KEY="<API_KEY>"`,
+    ].join('\n'),
```

Comment:
The env variables names will change with #1236, but I suggest we keep them in this PR as they're for now. We just have to keep in mind that this file has to be updated in #1236.

Reply:
I havent changed any env var names just updated the script that generates the .env.example to work with what we currently have in the .env.example file itself. And added the new 'name'_API_KEY var